services/src/contexts/schema.sql

```diff
@@ -50,6 +50,109 @@ CREATE TYPE "TimeStep" AS (
     step OID
 );
 
+CREATE TYPE "DatasetName" AS (namespace text, name text);
+
+CREATE TYPE "Provenance" AS (
+    citation text,
+    license text,
+    uri text
+);
+
+CREATE DOMAIN "RgbaColor" AS smallint [4] CHECK (
+    0 <= ALL(value) AND 255 >= ALL(value)
+);
+
+CREATE TYPE "Breakpoint" AS (
+    "value" double precision,
+    color "RgbaColor"
+);
+
+CREATE TYPE "DefaultColors" AS (
+    -- default
+    default_color "RgbaColor",
+    -- over/under
+    over_color "RgbaColor",
+    under_color "RgbaColor"
+);
+
+CREATE TYPE "ColorizerType" AS ENUM (
+    'linearGradient', 'logarithmicGradient', 'palette', 'rgba'
```

Comment:
The other enum variants in this files are capitalized. Should we do this uniformly?

Reply:
[a28413d](https://github.com/geo-engine/geoengine/pull/844/commits/a28413d25a68ac5b2eabdf4c1515ef267b60a5e6)